harcownik-frontend/src/components/userRequests/UserRanksFunction.jsx

```diff
@@ -2,24 +2,33 @@
 import axios from "axios";
 
 import "./UserRequests.css";
+import authHeader from "../../api/authHeader";
 
 export const getLevel = async () => {
   try {
-    const response = await axios.get("http://localhost:8000/level_reports");
+    const response = await axios.get(
+      "http://localhost:8000/level_reports",
+      authHeader()
+    );
+    console.log("Rank response:", response.data);
     return response.data;
   } catch (error) {
     console.error(error);
   }
 };
 
-export const postLevelRaports = async (level) => {
+export const postLevelRaports = async (level, userID) => {
```

Comment:
Please use good programming principles. variable should start from camelcase but in next word should be first letter uppercase (just first).

Reply:
Please change in each place where you use this variable